src/Microsoft.VisualStudio.AppDesigner/PropPages/PropertyControlData.vb

```diff
@@ -1537,9 +1538,24 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             Debug.Assert(Extender IsNot Nothing)
 
             'Note: For property get we must pass in the extended object as the component.
-            Return Descriptor.GetValue(Extender)
+            Return GetValue(Descriptor, Extender)
```

Comment:
I don't understand thi change - why do we need a CPS dependency in the prop page? This is the same for a regular csproj - would this new code cause problems for a desktop project?